src/fat.rs

```diff
@@ -69,44 +68,48 @@ where
         ))
     }
 
-    // Get's the [OpType] of the `FatNode`.
+    /// Get's the [OpType] of the `FatNode`.
     pub fn get(&self) -> &'c OT {
         self.hugr.get_optype(self.node).try_into().ok().unwrap()
     }
 }
 
 impl<'c, OT, H> FatNode<'c, OT, H> {
-    // Get's the [Node] of the `FatNode`.
+    /// Get's the [Node] of the `FatNode`.
     pub fn node(&self) -> Node {
         self.node
     }
 
-    // Get's the [HugrView] of the `FatNode`.
+    /// Get's the [HugrView] of the `FatNode`.
     pub fn hugr(&self) -> &'c H {
         self.hugr
     }
 }
 
 impl<'c, H: HugrView + ?Sized> FatNode<'c, OpType, H> {
-    // Creates a new general `FatNode` from a [HugrView] and a [Node].
-    //
-    // Panics if the node is not valid in the [Hugr].
+    /// Creates a new general `FatNode` from a [HugrView] and a [Node].
+    ///
+    /// Panics if the node is not valid in the [Hugr].
     pub fn new_optype(hugr: &'c H, node: Node) -> Self {
         assert!(hugr.valid_node(node));
         FatNode::new(hugr, node, hugr.get_optype(node))
     }
 
-    // Tries to downcast a genearl `FatNode` into a specific `OT`.
+    /// Tries to downcast a genearl `FatNode` into a specific `OT`.
     pub fn try_into_ot<OT: 'c>(&self) -> Option<FatNode<'c, OT, H>>
     where
         &'c OpType: TryInto<&'c OT>,
     {
         FatNode::try_new(self.hugr, self.node)
     }
 
-    // Creates a specific `FatNode` from a general `FatNode`.
-    //
-    // Panics if the node's `get_optype` is not `OT`.
+    /// Creates a specific `FatNode` from a general `FatNode`.
+    ///
+    /// Panics if the node is not valid in the `Hugr` or if it's `get_optype` is
+    /// not an `OT`.
+    ///
+    /// Note that while we do check that the type of the node's `get_optype`, we do
```

Comment:
This sentence seems incomplete; something missing after "while we do check that the type of the node's get_optype".

Reply:
> I think it is complete, continued on the following line.
> 
> "Note that while we do check that the type of the node's `get_optype`, we do not verify that it is actually equal to `ot`.:

This isn't grammatical -- perhaps you want to delete the first "that"?